aibolit/model/model.py

```diff
@@ -151,4 +148,73 @@ def predict(self, X, quantity_func='log', th=1.0):
             except Exception:
                 raise Exception("Unknown func")
 
-        return (np.array(ranked), pairs[:, 0].T.tolist()[::-1])
+        if not return_acts:
+            return (np.array(ranked), pairs[:, 0].T.tolist()[::-1])
+        return np.array(ranked), pairs[:, 0].T.tolist()[::-1], np.zeros(X.shape[0]) - 1
+
+    def get_array(self, X, mask, i, incr):
+        """
+        Args:
+            X: np.array with shape (number of snippets, number of patterns).
+            mask: np.array with shape (number of snippets, number of patterns).
+            i: int, 0 <= i < number of patterns.
+            add: bool.
+        Returns:
+            X1: modified np.array with shape (number of snippets, ).
+        """
+
+        X1 = X.copy()
+        X1[:, i][mask[:, i]] += incr
+
+        return X1
+
+    def get_minimum(self, c1, c2, c3):
+        """
+        Args:
+            c1, c2, c3: np.array with shape (number of snippets, ).
+        Returns:
+            c: np.array with shape (number of snippets, ) -
+            elemental minimum of 3 arrays.
+            number: np.array with shape (number of snippets, ) of
+            arrays' numbers with minimum elements.            .
+        """
+
+        c = np.vstack((c1, c2, c3))
+
+        return np.min(c, 0), np.argmin(c, 0)
+
+    def informative(self, X, return_acts=False):
```

Comment:
what is the return_acts parameter?

Reply:
If true, return acts, needed to do with most important pattern (np changes, -1, +1)